packages/bento-frontend/src/pages/caseDetail/FilesView/FilesTableView.js

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from 'react';
+import { Container, withStyles } from '@material-ui/core';
+import {
+  filesTable,
+} from '../../../bento/caseDetailData';
+import { Wrapper } from '../../../bento-core/Wrapper';
+import { themeConfig, wrapperThemConfig } from './Theme';
+import { TableView, TableContext } from '../../../bento-core/PaginationTable';
+import { footerConfig } from './Wrapper';
+
+const FilesTableView = ({
+  classes,
+  data,
+}) => {
+  const initTblState = (initailState) => ({
+    ...initailState,
```

Comment:
@rana22 Correct me if I am wrong, But Shouldn’t we spread initailState prop at the very end of the object?